### PR TITLE
Update icon and logo

### PR DESCRIPTION
**Pull request by VictoriaBeilsten-Edmands**

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -1,21 +1,16 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "Davidia",
+  "name": "Davidia",
   "icons": [
     {
       "src": "favicon.ico",
```
Comment (PeterC-DLS) [inline public/manifest.json:6]: You haven't replaced the favicon with https://www.diamond.ac.uk/dam/Images/icons/favicon.ico

Comment (VictoriaBeilsten-Edmands) [reply public/manifest.json:6]: favicon.ico is a different file with the same name.

Comment (PeterC-DLS) [reply public/manifest.json:6]: Ah, yes, I will mislead by the red and missed the 21% reduction. Note this new `.ico` is only 16x16.

File: public/manifest.json [hunk 2]
```diff
@@ -1,21 +1,16 @@
...
       "sizes": "64x64 32x32 24x24 16x16",
```
Comment (PeterC-DLS) [inline public/manifest.json:7]: Needs updating (see my last comment)

**Review by PeterC-DLS — APPROVED** [review]
Looks fine to me

**Pull request merged**